rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -329,6 +337,40 @@ def _decode_repay(
 
         return return_event, repay_event
 
+    def _decode_mint_events(self, context: DecoderContext) -> None:
```

Comment:
After reading the code and your comment I understand this needs a rename

```suggestion
    def _maybe_decode_mint_events(self, context: DecoderContext) -> None:
```